component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/IdentityDAOImpl.java

```diff
@@ -332,26 +339,66 @@ public int getSize() throws Exception {
     }
   }
 
+  private Query getIdentitiesQueryCount(String providerId) {
+    // Oracle and MSSQL support only 1/0 for boolean, Postgresql supports only
+    // TRUE/FALSE, MySQL supports both
+    String dbBoolFalse = isOrcaleDialect() || isMSSQLDialect() ? "0" : "FALSE";
+    String dbBoolTrue = isOrcaleDialect() || isMSSQLDialect() ? "1" : "TRUE";
```

Comment:
No more needed as we do not support Oracle and MSSAL server .

Reply:
Removed !